backend/app/queries/spree/admin/reports/average_order_values_query.rb

```diff
@@ -33,8 +33,15 @@ def group_by_date
         end
 
         def apply_date_filters(scope)
-          scope = scope.where('completed_at >= ?', opts[:completed_at_min]) if opts[:completed_at_min]
-          scope = scope.where('completed_at <= ?', opts[:completed_at_max]) if opts[:completed_at_max]
+          if opts[:completed_at_min]
+            min = Time.zone.parse(opts[:completed_at_min]).beginning_of_day
```

Comment:
shouldn't we use `Time.current` ?

Reply:
nope, we can't do `Time.current.parse('...')`